developer-docs-site/docs/index.md

```diff
@@ -10,7 +10,7 @@ hide_table_of_contents: true
 
 Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
 
-## Start here
+## Set up the Aptos environment and take the tutorials
```

Comment:
Maybe `try the tutorials`.  Idk why `take` rubs me the wrong way.  English is hard

Reply:
I used "start with the tutorials" since this is the first set of docs in the list. WDYT?